src/query-builder/function-module.ts

```diff
@@ -60,14 +60,15 @@ export class FunctionModule<DB, TB extends keyof DB> {
    * Calls the `avg` function for the column given as the argument.
    *
    * If this is used in a `select` statement the type of the selected expression
-   * will be `number | string` by default. This is because Kysely can't know the
-   * type the db driver outputs. Sometimes the output can be larger than the
-   * largest javascript number and a string is returned instead. Most drivers
-   * allow you to configure the output type of large numbers and Kysely can't
-   * know if you've done so.
+   * will be `number | string | null` by default. This is because Kysely can't know
```

Comment:
This isn't true is it? The default is now `number | string` with a user opt-in of including the `null`?

Reply:
Good catch! forgot about aligning the docs.. 😢